backend/ingestion/extract_census_data.py

```diff
@@ -1 +1,147 @@
- 
\ No newline at end of file
+import os
+import sys
+import csv
+import logging
+import requests
+from typing import List, Dict
+from dotenv import load_dotenv
+
+########################################################################################
+# SETUP
+########################################################################################
+
+# Enviornment variables
+load_dotenv()
+CENSUS_API_KEY = os.getenv("CENSUS_API_KEY")
```

Comment:
when you get a chance please add the API key here https://uchicago.app.box.com/notes/1509025578294 so we can add it to our `.env` files! Thank you!